web/client-api/src/main/java/io/deephaven/web/client/api/JsTable.java

```diff
@@ -1220,6 +1230,20 @@ public void handleDelta(ClientTableState current, DeltaUpdates updates) {
                 nonViewportSub.handleDelta(updates);
                 return;
             }
+            if (isStreamTable) {
```

Comment:
This whole block should be able to move upstream too, so that if there is more than one Table that is attached to a single ticket, they all get their removes deferred at once, instead of each tracking its own deferrals?

ClientTableState.handleDelta is before it fans out to each downstream sub (admittedly it is weird to hold state about a subscription there), or WorkerConnection.incrementalUpdates is one step upstream of that. I could be pretty easily convinced that neither is appropriate.

Upstream another step is the anonymous class that handles FlightData messages arriving - this _does_ track state of messages, presently to be able to see how many more messages are needed before the current delta is done. This use case smells a bit like that?